packages/expo-modules-core/ios/Swift/Networks/ExpoRequestCdpLogger.swift

```diff
@@ -0,0 +1,70 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+import Foundation
+
+/**
+ The `ExpoNetworkInterceptorProtocolDelegate` implementation to dispatch CDP (Chrome DevTools Protocol) events
+ */
+@objc
+public final class ExpoRequestCdpLogger: NSObject, ExpoRequestInterceptorProtocolDelegate {
+  private var delegate: ExpoRequestCdpLoggerDelegate?
+  internal var dispatchQueue = DispatchQueue(label: "expo.requestCdpLogger")
```

Comment:
Shouldn't it be a static var (or even top-level)?

Reply:
the whole `ExpoRequestCdpLogger` class is a singleton already. it looks like moving `dispatchQueue` to static is redundant. does that make sense to you?